SourceryRuntime/Sources/TypeName.swift

```diff
@@ -66,23 +66,24 @@ public final class TypeName: NSObject, AutoCoding, AutoEquatable, AutoDiffable,
             self.isOptional = false
         } else {
             name = name.bracketsBalancing()
+            let trimmedInoutName = name.trimmingPrefix("inout ")
```

Comment:
I think we don't need `let trimmedInoutName` and yes we can just trim `inout` and whitespace. You can check 26019c3